README.md

```diff
@@ -88,6 +104,12 @@ all:
         rack02.example.com:
 ```
 
+Note: the pacmaker role requires host-specifc variables that should be defined in the hosts file, they are as follows:
```

Comment:
Note: the Pacemaker role requires host-specific variables that should be defined in the hosts file, they are as follows:

Reply:
Would it be worth adding required variables to the example hosts file?